source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -222,15 +222,14 @@ namespace irr
 {
 namespace video
 {
-
 // -----------------------------------------------------------------------
 // WINDOWS CONSTRUCTOR
 // -----------------------------------------------------------------------
 #ifdef _IRR_COMPILE_WITH_WINDOWS_DEVICE_
 //! Windows constructor and init code
 COpenGLDriver::COpenGLDriver(const irr::SIrrlichtCreationParameters& params,
 		io::IFileSystem* io, CIrrDeviceWin32* device, const asset::IGLSLCompiler* glslcomp)
-: CNullDriver(device, io, params), //COpenGLFunctionTable(),   // What to do here?
+: CNullDriver(device, io, params), COpenGLFunctionTable(),
```

Comment:
why does `COpenGLDriver` inherit from `COpenGLFunctionTable` ....

Function table is supposed to be used by composition, not inheritance, and its supposed to be in the `SAuxContext` only